packages/psss-server/.env.example

```diff
@@ -0,0 +1,8 @@
+CLIENT_BASIC_AUTH_HEADER=
+DB_NAME=
+DB_PASSWORD=
+DB_URL=
+DB_USER=
```

Comment:
So in the end I guess we did need a db connection for the sake of session management huh? Makes me wonder if perhaps switching to a different system for session management might not be a bad idea? Perhaps a small file based db per app which just handles it's own sessions?

Obviously not for now, just a general thought 👍

Reply:
Yeah I actually looked into other options, including a file based db. They come with some risks that would need to be thought through though, and in the end I figured...I've got ready access to a nice database, I might as well use it for now. Notice though that the only thing passed around is the model for the psss_session table, so it kinda restricts it to have single-table access.